draft-ietf-quic-transport.md

```diff
@@ -6369,14 +6369,17 @@ endpoints that share a static key for stateless reset (see {{reset-token}}) MUST
 be arranged so that packets with a given connection ID always arrive at an
 instance that has connection state, unless that connection is no longer active.
 
+More generally, servers MUST NOT generate a stateless reset if a connection with
```

Comment:
So we choose to omit saying "_valid_ stateless reset", assuming this is implied?  I.e. a server may generate a stateless reset for a connection handled by a different server, but it is ok if they do not share stateless reset secret, since the resulting stateless reset will be invalid.

Reply:
Yes, I think that "valid" is implied.  In the case of one server generating a stateless reset based on a connection ID that came from another server, that wouldn't affect that other connection. It would only affect the connection that the server generating the stateless reset might have created, if there was such a connection.

Reply:
Ok.  Just to be clear, it is not about servers that generate connection IDs.  It is about _sets of servers that share the same Stateless Reset secrets_.  Servers MUST NOT send a stateless reset for a connection ID, if it is possible that this connection ID is active on any server that is still able to handle this connection and shares the same Stateless Reset secret.

Reply:
The following paragraph is a bit misleading in case of 5-tuple load balancers.  It is talking about routing of "**the** packet".  The specific packet might have a fixed routing due to its 5-tuple.  An attacker can rewrite the packet's source address to replace it with its own, causing the forged packet to be routed to a different server.  What matters is a possibility that routing of packets with this CID -- can their routing be affected by the attacker?